job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/publishImageGallery.groovy

```diff
@@ -0,0 +1,14 @@
+job('example-1') {
+  publishers {
+    publishImageGallery('some title', './some/path/with/regex/*.png')
+  }
+}
+
+job('example-2') {
+  publishers {
+    publishImageGallery('some title', './some/path/with/regex/*.png') {
+      imageWidthText '15'
```

Comment:
Use the UI labels as guides for naming methods. Change this to `imageWidth`. And change the argument to `Integer` since this is a numeric value. And add the round brackets around the arguments. All examples use brackets and this should be consistent.